src/style.css

```diff
@@ -0,0 +1,32 @@
+*,
+html {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  background-color: #fff;
```

Comment:
Remove redundant styles

Reply:
thanks for the comments! 
I see that PR already merged. Should I still adjust OR these are the comments for the future?